### DISCUSSION TITLE
Extensions defined in devcontainer.json are not installed

### DISCUSSION THREAD

**Discussion by maiertech** [community/community#25003] [Codespaces]
<p>I have just noticed that in newly created codespaces plugins defined in <code>devcontainer.json</code> under the <code>extensions</code> prop are not installed. I have not created a new Codespace in a few days, so I am not sure when exactly the problem started.</p>
<p>Here is a screenshot that shows the missing extensions:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/f/3f20b614826d41246734343f7001a13af2a911f4.png" data-download-href="/uploads/short-url/90sab46D0gGrbg64pNYeVX3XQgc.png?dl=1" title="Screen Shot 2021-01-19 at 2.26.51 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/1482402/181088112-86f5742c-257e-4f57-a126-ef16a6634878.png" alt="Screen Shot 2021-01-19 at 2.26.51 PM" data-base62-sha1="90sab46D0gGrbg64pNYeVX3XQgc" width="690" height="280" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f20b614826d41246734343f7001a13af2a911f4_2_690x280.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f20b614826d41246734343f7001a13af2a911f4_2_1035x420.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f20b614826d41246734343f7001a13af2a911f4_2_1380x560.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/f/3f20b614826d41246734343f7001a13af2a911f4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-01-19 at 2.26.51 PM</span><span class="informations">1910×776 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by arcdigital** [comment]
<p>I’m seeing the same issue. None of the ones in my devcontainer are getting installed (but the ones synced via my settings sync do get installed). I also noticed that the code cli returns [object Object] for many commands including --list-extensions and --install-extension {extensionid}</p>

**Comment by jkeech** [comment, marked as answer]
<p>Sorry for this issue. It looks like there is a bug in the recent VS Code insiders releases that breaks the command line install of extensions, which Codespaces uses to install extensions from devcontainer.json. We’ve rolled back to an older VS Code commit for now. Unfortunately, it won’t fix existing codespaces automatically, but all new codespaces should install devcontainer.json extensions. Let us know if you are hitting this issue on new codespaces. Thanks!</p>

**Comment by maiertech** [comment]
<p>Yes, in newly created codespaces it seems fine.</p>

**Comment by sunpengRvi** [comment]
<p>Could you please give the issue number or link which I can refer to? Thanks</p>
